--- configure.ac	2009-06-24 03:27:42.000000000 +0200
+++ configure.ac.oden	2009-12-26 17:19:52.221287952 +0100
@@ -42,11 +42,8 @@
 [GCC_WARN_FLAGS=""
 CFLAGS=""
 if test x"$GCC" = "xyes"; then
-    GCC_WARN_FLAGS="-Wall -Waggregate-return -Wcast-align -Wchar-subscripts -Wcomment -Wformat \
-        -Wimplicit -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wno-long-long \
-        -Wparentheses -Wpointer-arith -Wredundant-decls -Wreturn-type -Wswitch -Wtrigraphs \
-        -Wuninitialized -Wunused -Wwrite-strings -Wshadow -Wstrict-prototypes -Wcast-qual"
-    CFLAGS="-O3 $GCC_WARN_FLAGS"
+    GCC_WARN_FLAGS=""
+    CFLAGS="$GCC_WARN_FLAGS"
 fi]
 AC_SUBST(GCC_WARN_FLAGS)
 AC_SUBST(CFLAGS)
